app/components/behandeling-van-agendapunt.hbs

```diff
@@ -1,102 +1,87 @@
-  </AuHeading>
-  <div class="au-o-box au-o-box--small au-c-meeting-chrome-card au-u-margin-bottom-none">
-    <AuHeading @level="4" @skin="6" class="au-c-meeting-chrome-card__title au-c-meeting-chrome-card__title--inset au-u-margin-bottom-small">{{t "behandelingVanAgendapunten.visibility"}}</AuHeading>
-    <AuLabel><Input @type="checkbox" @checked={{this.openbaar}} {{on "input" this.toggleOpenbaar}} /> &nbsp;{{t "behandelingVanAgendapunten.openbaarLabel"}}</AuLabel>
+  <div class="au-u-margin-top au-u-margin-bottom">
+    <AuPill @skin="warning" @icon="alert-triangle" @iconAlignment="left">{{t
+            "behandelingVanAgendapunten.needsParticipants"}}</AuPill>
   </div>
+{{/if}}
 
-  {{#if this.fetchParticipants.isRunning}}
-    <AuLoader @padding="small" />
-    <AuHelpText>{{t "participationList.loadingTitle"}}</AuHelpText>
-  {{else}}
-    <ParticipationList
-      @chairman={{this.voorzitter}}
-      @secretary={{this.secretaris}}
-      @defaultSecretary={{@defaultSecretary}}
-      @defaultChairman={{@defaultChairman}}
-      @bestuursorgaan={{@bestuursorgaan}}
-      @possibleParticipants={{@possibleParticipants}}
-      @aanwezigenBijStart={{this.aanwezigen}}
-      @afwezigenBijStart={{this.afwezigen}}
-      @onSave={{this.saveParticipants}}
-      @modalTitle={{t "behandelingVanAgendapunten.participationListButton"}}
-      @meeting={{@meeting}}
-      @readOnly={{or this.published @readOnly}}
-   />
-  {{/if}}
-    {{#if this.hasParticipants}}
-      <div class="au-c-meeting-chrome-card au-u-margin-bottom-none">
-        <AuHeading @level="4" @skin="6" class="au-c-meeting-chrome-card__title">{{t "behandelingVanAgendapunten.votingTitle"}}</AuHeading>
-        <Treatment::Voting @bestuursorgaan={{@bestuursorgaan}} @behandeling={{@behandeling}}/>
-      </div>
-    {{else}}
-    <div class="au-u-margin-top au-u-margin-bottom">
-      <AuPill @skin="warning" @icon="alert-triangle" @iconAlignment="left">{{t "behandelingVanAgendapunten.needsParticipants"}}</AuPill>
-    </div>
-    {{/if}}
-  {{#if this.tryToFetchDocument.lastSuccessful}}
-    <div class="au-c-meeting-chrome-card au-c-meeting-chrome-card--editor au-u-margin-bottom-small au-u-margin-top-small">
-      <AuHeading @level="4" @skin="6" class="au-c-meeting-chrome-card__title au-u-margin-bottom">{{t "behandelingVanAgendapunten.contentTitle"}}</AuHeading>
-      <BesluitDocumentContainer @document={{this.document}} @handleRdfaEditorInit={{this.handleRdfaEditorInit}} @saving={{this.saveEditorDocument.isRunning}} @editor={{this.editor}}/>
-    </div>
-  {{/if}}
+
+<ReadonlyTextBox @title={{t "behandelingVanAgendapunten.contentTitle"}}>
+  {{@behandeling.documentContainer.currentVersion.htmlSafeContent}}
+</ReadonlyTextBox>
+{{#if this.editable}}
   <AuButtonGroup>
-    <AuButton {{on "click" this.save}} @disabled={{if this.saveEditorDocument.isRunning true false}}>
-      {{t "behandelingVanAgendapunten.saveButton"}}
+    <AuButton {{on "click" this.goToEdit}}>
```

Comment:
this could just be a LinkTo perhaps ?

Reply:
at the time the version of AU we have didnt support models with multiple params in AuLink
I see that theres been a new release so Ill try it out

Reply:
FYI I think you only need to pass the treatment id since the router already has the meeting id.

Reply:
Hmm since it's a component that in theory could be put anywhere passing everything might make sense actually.

Reply:
at some point ember complained so I just assumed I had to put everything everywhere
but it does seem to work with only 1 argument

Reply:
Visually the AuLink looks very different than the button though, and with the changes dieter made I think the button looks more uniform (it's what we use for the other, non-fullpage modals)

Reply:
I would just use a `LinkTo` and add the needed classes. I think Appuniversum should simply provide a component that makes an `<a>` tag look like a button since it's such a common use-case. I already created an issue for that: https://github.com/appuniversum/ember-appuniversum/issues/110 :smile: 

Using button for navigation just because they look like buttons seems weird to me :smile:.

Reply:
well yeah
but this story also need to get done at some point :stuck_out_tongue:

Reply:
I've created a story on the backlog, so we can take this up at a later time